sklearn/ensemble/tests/test_iforest.py

```diff
@@ -295,6 +295,28 @@ def test_score_samples():
                        clf2.score_samples([[2., 2.]]))
 
 
+@pytest.mark.filterwarnings('ignore:default contamination')
+@pytest.mark.filterwarnings('ignore:behaviour="old"')
+def test_iforest_warmstart():
+    """Test iterative addition of iTrees to an iForest """
+
+    rng = check_random_state(0)
+    X = iris.data
```

Comment:
I think you can even use a smaller data set as we are not testing the performance here
```suggestion
    X = rng.randn(20, 2)
```

Reply:
true